doc/classes/BoneMap.xml

```diff
@@ -40,6 +40,9 @@
 	<members>
 		<member name="profile" type="SkeletonProfile" setter="set_profile" getter="get_profile">
 			A [SkeletonProfile] of the mapping target. Key names in the [BoneMap] are synchronized with it.
+
+			
+			[b]Note:[/b] this property is not accessible to import scripts. The [SkeletonProfile] is only accessible to the UI during import.
```

Comment:
```suggestion
			[b]Note:[/b] This property is not accessible to import scripts. The [SkeletonProfile] is only accessible to the UI during import.
```
I missed this

You also need to remove the two empty lines above, they aren't necessary and adds two empty paragraphs